client/blocks/plan-storage/docs/example.jsx

```diff
@@ -46,20 +48,34 @@ export default React.createClass( {
 					<PlanStorage siteId={ siteId } />
 				</div>
 				<div style={ { marginBottom: 16 } }>
-					<PlanStorageButton
-						sitePlanName={ plans.free }
+					<PlanStorageBar
+						siteSlug={ primarySite.slug }
+						sitePlanSlug={ PLAN_FREE }
 						mediaStorage={ mediaStorage.green }
 					/>
 				</div>
 				<div style={ { marginBottom: 16 } }>
-					<PlanStorageButton
-						sitePlanName={ plans.free }
+					<PlanStorageBar
+						siteSlug={ primarySite.slug }
+						sitePlanSlug={ PLAN_PERSONAL }
 						mediaStorage={ mediaStorage.yellow }
 					/>
 				</div>
 				<div style={ { marginBottom: 16 } }>
-					<PlanStorageButton
-						sitePlanName={ plans.premium }
+					<PlanStorageBar
+						siteSlug={ primarySite.slug }
+						sitePlanSlug={ PLAN_PREMIUM }
+						mediaStorage={ mediaStorage.red }
+					/>
+				</div>
+
+				<div style={ { marginBottom: 16 } }>
+					<span style={ { fontSize: 12, color: 'grey' } }>
+						No render PlanStorage here since it's a business plan.
```

Comment:
How about `Business plans have unlimited storage, so PlanStorage will not be rendered.`